### DISCUSSION TITLE
What is the difference between the official and unstructured git reference for detectron2?

### DISCUSSION THREAD

**Discussion by codingbutstillalive** [Unstructured-IO/unstructured#525] [Q&A] [answered]
The official just relates to FAIR's detectron2 repo, while unstructured uses some specific egg-info, which isn't clear to me. Is it needed for the layout-parsing behavior of the model?

Also, this egg-reference seems to be outdated and doesn't work.

**Comment by MthwRobinson** [comment, marked as answer]
Hi @codingbutstillalive  - the most recent tag resulted in a dependency conflict. We've pinned `detectron2` to a commit that's more recent than the latest tag to avoid that.

**Reply by codingbutstillalive** [reply]
Thank you!
